spec/mailers/validation_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe ValidationMailer, type: :mailer do
```

Comment:
Nice, looks good. I set up webmock and vcr on another branch so we will may have to add vcr to these tests after we merge.